api/HTMLMenuElement.json

```diff
@@ -165,7 +165,7 @@
               "version_added": "8"
             },
             "ie": {
-              "version_added": true
+              "version_added": "≤6"
```

Comment:
This change seems a little bit suspect, why is only api.HTMLMenuElement.type updated but not the parent entry or api.HTMLMenuElement.compact? Don't the tests cover those?